src/specific/display.c

```diff
@@ -10,10 +10,66 @@
 
 #include <stdlib.h>
 
+// The screen resolution is controlled by two variables are indices
+// within an array of predefined screen resolutions.
+// Actual screen resolution, sometimes different from the game
+// resolution (during FMVs, main menu sequence, static pictures etc.)
+static int32_t HiRes = 0;
+// The resolution to render the game in. This is what gets saved in
+// settings and the like.
+static int32_t GameHiRes = 0;
+
+bool SetGameScreenSizeIdx(int32_t idx)
+{
+    if (idx >= 0 && idx < RESOLUTIONS_SIZE) {
+        GameHiRes = idx;
+        return true;
+    }
+    return false;
+}
+
+bool SetPrevGameScreenSize()
+{
+    if (GameHiRes - 1 >= 0) {
```

Comment:
GameHiRes > 0

Reply:
which is just if GameHiRes

Reply:
True, but I wanted this condition to mirror that of `SetNextGameScreenSize`'s.